requirements.txt

```diff
@@ -3,6 +3,6 @@
 # Manually managing azure-functions-worker may cause unexpected issues
 
 azure-functions
-PyGithub[integrations]==1.54.1
+PyGithub[integrations]==1.58.2
```

Comment:
pyjwt and PyGithub were not compatible with eachother's versions, so I increased the PyGithub version.